umbracocms-webapp-with-storage-and-cache/azuredeploy.json

```diff
@@ -0,0 +1,576 @@
+{
+   "$schema":"http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
+   "contentVersion":"1.0.0.0",
+   "parameters":{
+      "siteName":{
+         "type":"string",
+         "defaultValue":"",
+         "metadata":{
+            "description":"Name of azure web app"
+         }
+      },
+      "hostingPlanName":{
+         "type":"string",
+         "metadata":{
+            "description":"Name of azure web app service plan"
+         }
+      },
+      "serverFarmResourceGroup":{
+         "type":"string",
+         "defaultValue":"",
+         "metadata":{
+            "description":"Name of your resource group"
+         }
+      },
+      "siteLocation":{
+         "type":"string",
```

Comment:
Can you have any Azure location here? If not then consider using allowed values to avoid deployment errors